### PR TITLE
Add marseel to sig-scalability-leads

### PR DESCRIPTION
**Pull request by mborsz**
/assign @wojtek-t 
/cc @marseel

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -101,6 +101,7 @@ aliases:
     - mm4tt
     - shyamjvs
     - wojtek-t
+    - marseel
```
Comment (wojtek-t) [inline OWNERS_ALIASES:104]: Let's remove @mm4tt and sort alphabetically :)

Comment (mborsz) [reply OWNERS_ALIASES]: Done

**Pull request merged**